### PR TITLE
Initial readme update

### PR DESCRIPTION
**Pull request by h1dden-da3m0n**
**Description**

Unfortunately I cannot really help with these points:

* The sample code is outdated.
* Should add a link to the sample(s)

However, I can help with the other two, so I opened this :wink:  

**Changes**

* update readme title
* update readme badges
* update readme headline hierarchy
* add `Reference Use-Cases` blurb to the end of readme

**Issues**

*  part of #107

### CODE REVIEW

**Comment by h1dden-da3m0n** [comment]
I hope I was not to aggressive in removing some of the badges and that the `Reference Use-Cases` is close to what you intended wit the blurb referring the Android and Android TV clients.
Anyway, happy to get any feedback on my changes :wink:

**Comment by h1dden-da3m0n** [comment]
On an unrelated note: I experimented a bit with moving the readme from Markdown to AsciiDoc (for no apparent reason) if you want to see the result have a look [h1dden-da3m0n/jellyfin-apiclient-java#test/adoc](https://github.com/h1dden-da3m0n/jellyfin-apiclient-java/tree/test/adoc)

**Comment by nielsvanvelzen** [comment]
Took a quick glance and it looks good so far. We should keep it markdown (although the TOC looks pretty cool!) because that's more common and known for contributors.

I'll read trough it properly later.

File: README.md [hunk 1]
```diff
@@ -160,3 +166,26 @@ apiClient.AuthenticateUserAsync("username", "password", new Response<Authenticat
...
 });
 ```
+
+---
+
+## Reference Use-Cases
+
+This API client library can be utilised in any Java or Kotlin based client application and serves as an abstraction layer to interact with the API endpoints provided by a current version of Jellyfin server.
+The Jellyfin Project already uses this client library within two of its official clients and provides this utility to any other developer with the need for this library.
+You can use these reference use-cases to help in case you plan to develop your own JVM based Jellyfin client(s).
```
Comment (nielsvanvelzen) [inline README.md:176]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -160,3 +166,26 @@ apiClient.AuthenticateUserAsync("username", "password", new Response<Authenticat
...
+
+---
+
+## Reference Use-Cases
+
+This API client library can be utilised in any Java or Kotlin based client application and serves as an abstraction layer to interact with the API endpoints provided by a current version of Jellyfin server.
+The Jellyfin Project already uses this client library within two of its official clients and provides this utility to any other developer with the need for this library.
+You can use these reference use-cases to help in case you plan to develop your own JVM based Jellyfin client(s). 
+
+### Jellyfin Android
```
Comment (nielsvanvelzen) [inline README.md:178]:
```suggestion
### Jellyfin for Android
```

File: README.md [hunk 3]
```diff
@@ -160,3 +166,26 @@ apiClient.AuthenticateUserAsync("username", "password", new Response<Authenticat
...
+
+[Jellyfin Android](https://github.com/jellyfin/jellyfin-android) is the new official Kotlin based re-implementation of the old Android client.
+Should you intend to use this library with Kotlin as your language of choice, then you can use this client as a reference.
+
+### Jellyfin Android TV
```
Comment (nielsvanvelzen) [inline README.md:183]:
```suggestion
### Jellyfin for Android TV
```

File: README.md [hunk 4]
```diff
@@ -160,3 +166,26 @@ apiClient.AuthenticateUserAsync("username", "password", new Response<Authenticat
...
+
+## Reference Use-Cases
+
+This API client library can be utilised in any Java or Kotlin based client application and serves as an abstraction layer to interact with the API endpoints provided by a current version of Jellyfin server.
+The Jellyfin Project already uses this client library within two of its official clients and provides this utility to any other developer with the need for this library.
+You can use these reference use-cases to help in case you plan to develop your own JVM based Jellyfin client(s). 
+
+### Jellyfin Android
+
+[Jellyfin Android](https://github.com/jellyfin/jellyfin-android) is the new official Kotlin based re-implementation of the old Android client.
```
Comment (nielsvanvelzen) [inline README.md:180]:
```suggestion
[Jellyfin for Android](https://github.com/jellyfin/jellyfin-android) is our official Kotlin based Android client for phones and tablets.
```

File: README.md [hunk 5]
```diff
@@ -160,3 +166,26 @@ apiClient.AuthenticateUserAsync("username", "password", new Response<Authenticat
...
+Should you intend to use this library with Kotlin as your language of choice, then you can use this client as a reference.
```
Comment (nielsvanvelzen) [inline README.md:181]:
```suggestion

```

File: README.md [hunk 6]
```diff
@@ -160,3 +166,26 @@ apiClient.AuthenticateUserAsync("username", "password", new Response<Authenticat
...
+
+### Jellyfin Android TV
+
+[Jellyfin Android TV](https://github.com/jellyfin/jellyfin-androidtv) is the current official Android TV client.
```
Comment (nielsvanvelzen) [inline README.md:185]:
```suggestion
[Jellyfin for Android TV](https://github.com/jellyfin/jellyfin-androidtv) is the official Android TV client for devices running Android TV, Fire TV or Google TV.
```

File: README.md [hunk 7]
```diff
@@ -160,3 +166,26 @@ apiClient.AuthenticateUserAsync("username", "password", new Response<Authenticat
...
+It mostly represents a Java-based client using this library, yet is in the process of migrating to Kotlin.
```
Comment (nielsvanvelzen) [inline README.md:186]:
```suggestion

```

File: README.md [hunk 8]
```diff
@@ -160,3 +166,26 @@ apiClient.AuthenticateUserAsync("username", "password", new Response<Authenticat
...
+
+### Gelli
+
+[Gelli](https://github.com/dkanada/gelli) is a music-focused 3rd party Android client developed and maintained by one of the core Jellyfin members.
```
Comment (nielsvanvelzen) [inline README.md:190]:
```suggestion
[Gelli](https://github.com/dkanada/gelli) is a music-focused third-party Android client.
```

File: README.md [hunk 9]
```diff
@@ -160,3 +166,26 @@ apiClient.AuthenticateUserAsync("username", "password", new Response<Authenticat
...
+It represents a pure Java client utilising this client library.
```
Comment (nielsvanvelzen) [inline README.md:191]:
```suggestion
```

File: README.md [hunk 10]
```diff
@@ -160,3 +166,26 @@ apiClient.AuthenticateUserAsync("username", "password", new Response<Authenticat
 	}
 });
 ```
+
+---
+
+## Reference Use-Cases
```
Comment (nielsvanvelzen) [inline README.md:172]:
```suggestion
## Projects using the API client
```

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]
The code samples are outdated and definitely need to be changes, same for the installation (the "library" package is gone in the next release). I'll take care of those later.

File: README.md (hunk 10 above)
Comment (h1dden-da3m0n) [reply README.md:172]: I tried thinking of a proper headline and could not think of that xD

File: README.md [hunk 11]
```diff
@@ -1,12 +1,15 @@
...
 
 ---
 
 <p align="center">
 <img alt="Logo Banner" src="https://raw.githubusercontent.com/jellyfin/jellyfin-ux/master/branding/SVG/banner-logo-solid.svg?sanitize=true"/>
 <br/>
 <br/>
+<a href="https://dev.azure.com/jellyfin-project/jellyfin/_build?definitionId=24&_a=summary&repositoryFilter=3&branchFilter=257%2C257%2C257%2C257%2C257%2C257%2C257%2C257%2C257%2C257">
+<img alt="Azure DevOps builds" src="https://img.shields.io/azure-devops/build/jellyfin-project/7cce6c46-d610-45e3-9fb7-65a6bfd1b671/24.svg">
+</a>
```
[the remark this answers is not in the document]
Comment (h1dden-da3m0n) [reply README.md:12]: Done

**Review by nielsvanvelzen — APPROVED** [review]
Thanks! I'll make sure to update the last things before release :)

**Pull request merged**